brax/v2/base.py

```diff
@@ -335,12 +338,13 @@ class Box(Geometry):
   """
 
   halfsize: jp.ndarray
+  rgba: Optional[jp.ndarray] = None
 
 
 @struct.dataclass
 class Plane(Geometry):
   """An infinite plane whose normal points at +z in its coordinate space."""
-
+  rgba: Optional[jp.ndarray] = None
```

Comment:
I think we'll want a `= struct.field(pytree_node=False)` here

Reply:
With that in some of the tests I get `TypeError: __init__() missing 1 required positional argument: 'rgba'` it was bit tricky to me to find the way of adding it and getting no errors.

Reply:
Also the other tests in mjcf_test.py have `ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()` so I think  maybe  leaving it as None is the straight way to go unless the solution is undesirable.

Reply:
So the `TypeError: __init__() missing 1 required positional argument: 'rgba'` seems to be an error in barx.v2.geometry.mesh that needs resolving, at least on my branch. I'm not sure how you're getting the ValueError
Quite a bit changed in mjcf and base.System.geoms, and you're right that we don't want the pytree_node=False :)